logins-swift/Sync15Passwords/Sync15Logins/LoginsStorage.swift

```diff
@@ -80,7 +87,7 @@ open class LoginsStorage {
                 throw LockError.locked
             }
             try LoginsStoreError.unwrap({ err in
-                sync15_passwords_state_sync(engine,
+                sync15_passwords_sync(engine,
                                             unlockInfo.kid,
```

Comment:
Nit: alignment (maybe just put it on one line tbh, or indent everything by a tab or something)